src/index.ts

```diff
@@ -22,8 +22,21 @@ interface TelescopeNode {
   telescopicOut: TelescopicOutput;
 }
 
+interface Config {
+  separator?: string;
+  hoverable?: boolean;
```

Comment:
```suggestion
  /**
  * If true, allows sections to expand automatically on mouse over rather than requiring a click.
  */
  shouldExpandOnMouseOver?: boolean;
```

nit: making this property a little more verbose and descriptive

Reply:
👍 I'll rename throughout the file

If this is the level of fidelity we want for config options, because this update isn't backwards compatible regardless, should we rename 'separator' to 'inlineSeparator' or similar as well?

Reply:
@spencerc99 going to make a separate issue for the docs around `seperator` and `inlineSeperator`

Reply:
renamed !